ui/components/reservation-unit/Head.tsx

```diff
@@ -140,6 +142,13 @@ const Head = ({
   const pricing = getActivePricing(reservationUnit);
   const unitPrice = getPrice(pricing);
 
+  const unitPriceSuffix = useMemo(() => {
+    const suffix =
+      getPrice(pricing, undefined, false, true) !== "0" &&
```

Comment:
magic variables ahoy :D. Olis kiva jos nämä funktiot olisi tyypitettyjä niin olisi helppo päätellä mitä arvot tekevät.

Reply:
Onko tässä muuten useMemo tarpeen? että onko niin raskas laskutoimitus, kun kuitenkin tuolla dom:ssa tarkistettaan unitPrice &&, joten voisi vain hakea siellä tuon subventionSuffixin? näin on tehty toisellakin  sivulla tässä PR:ssä